src/contracts/interfaces/IBLSPubkeyRegistry.sol

```diff
@@ -66,7 +66,7 @@ interface IBLSPubkeyRegistry is IRegistry {
     function getApkForQuorum(uint8 quorumNumber) external view returns (BN254.G1Point memory);
 
     /// @notice Returns the index of the quorumApk index at `blockNumber` for the provided `quorumNumber`
-    function getApkIndicesForQuorumsAtBlockNumber(bytes calldata quorumNumbers, uint256 blockNumber) external view returns(uint32[] memory);
+    function getApkIndicesForQuorumsAtBlockNumber(bytes calldata quorumNumbers, uint256 blockNumber) external returns(uint32[] memory);
```

Comment:
presumably want to restore this to 'view' before merging this PR

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/88/commits/8888f2a31ff9e13bca8370846c35f1c48b45d48f